services/core-web/src/components/Forms/incidents/IncidentFormDocuments.tsx

```diff
@@ -70,7 +70,7 @@ const IncidentFormDocuments: FC<IncidentFormDocumentsProps> = ({
                 id={INITIAL_INCIDENT_DOCUMENTS_FORM_FIELD}
                 name={INITIAL_INCIDENT_DOCUMENTS_FORM_FIELD}
                 labelIdle='<strong>Drag & Drop your files or <span class="filepond--label-action">Browse</span></strong><br>
-              <div>Accepted filetypes: .kmz, .doc, .docx, .xlsx, .pdf</div>'
+              <div>Accepted filetypes: .kmz, .doc, .docx, .xlsx, .pdf, .jpeg, .tiff, .hiec</div>'
```

Comment:
Just noticed that .png isn't listed here! And should this include .msg or did that get taken out?

Reply:
.png and .msg was missed. Updated accordingly.